src/nvme/ioctl.c

```diff
@@ -262,16 +262,18 @@ enum nvme_cmd_dword_fields {
 	NVME_VIRT_MGMT_CDW10_RT_MASK				= 0x7,
 	NVME_VIRT_MGMT_CDW10_CNTLID_MASK			= 0xffff,
 	NVME_VIRT_MGMT_CDW11_NR_MASK				= 0xffff,
-	NVME_FORMAT_CDW10_LBAF_SHIFT				= 0,
+	NVME_FORMAT_CDW10_LBAFL_SHIFT				= 0,
```

Comment:
I fear we can't just rename definitions anymore. This breaks backwards compatibility.

Reply:
Fixed this - decided to use the old name for both cases to keep it backward compatible.